views/charts.tmpl

```diff
@@ -0,0 +1,252 @@
+                    'Ticket Pool Size Chart', true, false)
+                break;
+                
+                case "3": // pool value graph
+                    if (pool_value === undefined) { await resolvePromise(poolFunc); }
+                    drawGraph(pool_value, ["Date", "Ticket Pool Value"], true, 'Ticket Pool Value','Date', 
+                    'Ticket Pool Value Chart', true, false)
+                break;
+                
+                case "4": // block size graph
+                    if (block_size === undefined) { await resolvePromise(sizefunc);}
+                    drawGraph(block_size, ["Date", "Block Size"], false, 'Block Size', 'Date','Average Block Size Chart', true, false)
+                break;
+
+                case "5": // blockchain size graph
+                    if (blockchain_size === undefined) { await resolvePromise(sizefunc); }
+                    drawGraph(blockchain_size, ["Date", "BlockChain Size"], true, 'BlockChain Size', 'Date', 'BlockChain Size Chart', false, true)
+                break;
+
+                case "6":  // tx per block graph
+                    if (tx_per_block === undefined) { await resolvePromise(sizefunc); }
+                    drawGraph(tx_per_block, ["Date", "Number of Transactions Per Block"], false, 'Number of Transactions', 'Date',
+                     'Number of Transactions Per Block Chart', false, false)
+                break;
+
+                case "7": // tx per day graph
+                    if (tx_per_day === undefined) { await resolvePromise(txFunc);}
+                    drawGraph(tx_per_day, ["Date", "Number of Transactions Per Day"], true, 'Number of Transactions', 'Date', 
+                    'Number of Transactions Per Day Chart', true, false)
+                break;
+
+                case "8": // difficulty graph
+                    if (difficulty === undefined) { await resolvePromise(ticketsFunc);}
+                    drawGraph(difficulty, ["Date", "Difficulty"], true, 'Difficulty', 'Date', 'PoW Difficulty Chart', true, false)
+                break;
+                
+                case "9": // supply graph
+                    if (coin_supply === undefined ){ await resolvePromise(supplyFunc);}
+                    drawGraph(coin_supply, ["Date", "Coin Supply"], true, 'Coin Supply', 'Date', 'Total Coin Supply Chart', true, false)
+                break;
+
+                case "10": // block fee graph
+                    if (fee_per_block === undefined) { await resolvePromise(blockFeeFunc);}
+                    drawGraph(fee_per_block, ["Block Height", "Total Fee"], false, 'Total Fee (DCR)', 'Block Height', 
+                    'Total Fee Per Block Chart', true, false)
+                break;
+
+                case "11": // Duration between blocks graph
+                    if (time_btw_blocks === undefined) { await resolvePromise(sizefunc);}
```

Comment:
Please use camelCase for all client side naming. It's simply a convention amongst all major javascript codebases. So `time_btw_blocks` should be `timeBtwBlocks`

Reply:
Noted.